tools/debian/control

```diff
@@ -0,0 +1,78 @@
+Source: mgba
+Section: otherosfs
+Priority: extra
+Maintainer: Sérgio Benjamim <sergio_br2@yahoo.com.br>
+Build-Depends: cmake (>= 2.8.11),
+               debhelper (>= 9),
+               libavcodec-dev,
+               libavformat-dev,
+               libavresample-dev,
+               libavutil-dev,
+               libedit-dev,
+               libmagickwand-dev,
+               libpng-dev,
+               libqt5opengl5-dev,
+               libsdl2-dev,
+               libswscale-dev,
+               libzip-dev,
+               pkg-config,
+               qtbase5-dev,
+               qtmultimedia5-dev,
+               zlib1g-dev
+Standards-Version: 3.9.6
+Homepage: https://endrift.com/mgba/
+
+Package: libmgba
+Architecture: any
+Multi-Arch: same
+Depends: ${misc:Depends}, ${shlibs:Depends}
```

Comment:
There's no "libavcodec-extra-dev" package, it's up to the user install the libavcodec-extra